bucoffea/limit/limit.py

```diff
@@ -5,24 +5,42 @@
 from bucoffea.plot.util import acc_from_dir
 from bucoffea.limit.legacy import legacy_limit_input
```

Comment:
Just an additional quick comment: Should we change this legacy file that we are importing from? I think the relevant file is renamed now.

Reply:
Yes, this import it broken. Thanks for catching it!
I'm surprised flake8 didnt catch that...